packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -206,13 +206,13 @@ export class OrganizationAPI extends AbstractAPI {
   }
 
   public async getOrganizationMembershipList(params: GetOrganizationMembershipListParams) {
-    const { organizationId, limit, offset } = params;
+    const { organizationId, limit, offset, orderBy } = params;
```

Comment:
Any thoughts on automatically forwarding any params (outside of `organizationId`) the the query string? 

```suggestion
    const { organizationId, ...queryParams } = params;
```

Reply:
This was already being done for another existing method - I agree it's more extendable! 

If folks are using TS, then invalid params are caught in static types. For runtime, then BAPI is going to return a 400 error as well.